cogeo_mosaic/scripts/cli.py

```diff
@@ -321,8 +341,67 @@ def overview(
         output_profile,
         prefix,
         max_overview_level=overview_level,
+        method=method,
+        config=config,
+        threads=threads,
+        in_memory=in_memory,
+    )
+
+
```

Comment:
what?? `overview` now does nothing... It creates the config but doesn't use it for anything.

Reply:
I don't get what you are saying here @kylebarron